x-pack/plugins/maps/public/connected_components/map/mb/tooltip_control/tooltip_control.js

```diff
@@ -53,7 +53,7 @@ export class TooltipControl extends React.Component {
     });
   }
 
-  _getIdsForFeatures(mbFeatures) {
+  _getIdsAndFeaturesMeta(mbFeatures) {
```

Comment:
Remove meta from `_getIdsAndFeaturesMeta`. How about renaming to  `_getTooltipFeatures`?